src/Files.App/Services/ResourcesService.cs

```diff
@@ -46,9 +46,11 @@ public void SetAppThemeFontFamily(string contentControlThemeFontFamily)
 		public void SetCompactSpacing(bool useCompactSpacing)
 		{
 			var listItemHeight = useCompactSpacing ? 24 : 36;
+			var listItemMargin = useCompactSpacing ? "-2" : "0";
```

Comment:
Is there a way to do this without the quotation marks?

Reply:
I think this is the best option in C#.
The alternative is using something like:
`var listItemMargin = 0 - (2 * (int)useCompactSpacing)`